src/uct/ib/configure.m4

```diff
@@ -383,13 +380,14 @@ AS_IF([test "x$with_ib" = "xyes"],
            [AC_DEFINE([HAVE_TL_UD], 1, [UD transport support])])
 
        # XRQ with Tag Matching support
-       AS_IF([test "x$with_ib_hw_tm" != xno],
-           [AC_CHECK_HEADERS([infiniband/tm_types.h])
+       AS_IF([test "x$with_ib_hw_tm" != xno], [
             AC_CHECK_MEMBER([struct ibv_exp_tmh.tag], [with_ib_hw_tm=exp], [],
                             [[#include <infiniband/verbs_exp.h>]])
-            AC_CHECK_MEMBER([struct ibv_tmh.tag], [with_ib_hw_tm=upstream], [],
-                            [[#include <infiniband/tm_types.h>]])
-           ])
+            AS_IF([test "x$with_mlx5_dv" = "xyes"], [
+                AC_CHECK_HEADERS([infiniband/tm_types.h])
```

Comment:
can remove?

Reply:
it adds HAVE_INFINIBAND_TM_TYPES_H
later we use it create upstream/exp compatible headers
https://github.com/openucx/ucx/blob/master/src/uct/ib/rc/accel/rc_mlx5_common.h#L26